pkg/audit/disqualification_test.go

```diff
@@ -136,9 +137,11 @@ func TestDisqualifiedNodesGetNoDownload(t *testing.T) {
 
 		encScheme := upl.GetConfig(satellite).GetEncryptionScheme()
 		cipher := encScheme.Cipher
-		encryptedAfterBucket, err := streams.EncryptAfterBucket(ctx, "testbucket/test/path", cipher, &storj.Key{})
+		store := encryption.NewStore()
+		store.SetDefaultKey(new(storj.Key))
+		encryptedPath, err := encryption.EncryptPathWithStore("testbucket", paths.NewUnencrypted("test/path"), cipher, store)
 		require.NoError(t, err)
-
+		encryptedAfterBucket := "testbucket/" + encryptedPath.Raw()
```

Comment:
It would be better to ditch the `encryptedAfterBucket` var and modify the next line with `JoinPaths` appropriately.